wakuv2/filter_manager.go

```diff
@@ -0,0 +1,360 @@
+package wakuv2
+
+import (
+	"context"
+	"crypto/rand"
+	"errors"
+	"math/big"
+	"sync"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/libp2p/go-libp2p/core/peer"
+
+	"github.com/status-im/status-go/wakuv2/common"
+
+	node "github.com/waku-org/go-waku/waku/v2/node"
+	"github.com/waku-org/go-waku/waku/v2/protocol"
+	"github.com/waku-org/go-waku/waku/v2/protocol/filter"
+	"github.com/waku-org/go-waku/waku/v2/protocol/relay"
+	"github.com/waku-org/go-waku/waku/v2/protocol/subscription"
+	"go.uber.org/zap"
+	"golang.org/x/exp/maps"
+)
+
+const (
+	FilterEventAdded             int = 0
```

Comment:
You can use `iota` for this, also, generally you don't want to start from `0`, since in go is hard to tell whether something is not initialized or actually set to 0, if it's a non nullable value like in this case.

Reply:
Ah, yes, thanks.